aten/src/ATen/native/cuda/ForeachUnaryFunctors.cu

```diff
@@ -0,0 +1,204 @@
+#include <ATen/Dispatch.h>
```

Comment:
Do you plan to create a header for this and split it out into a separate file?

That way I could use UnaryOpFunctor as a standalone for my own custom ops.